comixed-library/src/main/java/org/comixed/adaptors/archive/SevenZipArchiveAdaptor.java

```diff
@@ -69,8 +69,11 @@ private void addFileToArchive(SevenZOutputFile archive, String filename, byte[]
     archive.write(content);
     archive.closeArchiveEntry();
 
-    this.log.debug("Cleaning up the temporary file");
-    tempFile.delete();
+    if (tempFile.delete()) {
+      this.log.debug("Cleaning up the temporary file");
```

Comment:
lombok generate this logger : 
private static final Logger log = org.apache.logging.log4j.LogManager.getLogger(SevenZipArchiveAdaptor.class);
so I think we can access to the log without the "this".

Reply:
Could you do a global search-and-replace to replace "this.log" in the Java code with "log" and submit a PR?

Reply:
Yes i can.